.github/actions/install-win-store-python/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Install Windows Store Python
+description: Installs Python from the Windows Store
+
+inputs:
+  python-version:
+    description: 'Python version to install'
+    default: '3.11'  # should default to the latest Python
```

Comment:
"3.X" will give you the latest stable release (3.11.1 at present)

Reply:
I'm not sure I'm following...given we need to manually specify the specific python we want somewhere, using `3.X` here would just require a translation later, right?

Reply:
My mistake - I read this as a question about "how can we default to the latest python". I guess it might be nice to support "3.X" syntax, but I wouldn't consider that a blocker by any means.